neuro_extras/image_builder.py

```diff
@@ -48,6 +52,7 @@ def __init__(
         self._client = client
         self._other_clients_configs = list(other_clients_configs)
         self._verbose = verbose
+        self.logger = logging.getLogger(__name__)
```

Comment:
minor: is there a reason for logger not to be file-global?

Reply:
that was just my wrong personal habit.
didn't know about the convention to define a logger at the top of every module that requires logging.
fixed in c23c62a
Thanks for pointing out 👍